src/main/kotlin/no/nav/dagpenger/oppslag/joark/JoarkClientSoap.kt

```diff
@@ -0,0 +1,14 @@
+package no.nav.dagpenger.oppslag.joark
+
+import no.nav.tjeneste.virksomhet.behandleinngaaendejournal.v1.binding.BehandleInngaaendeJournalV1
+import no.nav.tjeneste.virksomhet.behandleinngaaendejournal.v1.meldinger.FerdigstillJournalfoeringRequest
+
+class JoarkClientSoap(private val inngåendJournal: BehandleInngaaendeJournalV1) {
```

Comment:
minor - glemt en 'e' i 'inngåendJournal'. Forøvrig syntes jeg dette blir rart uansett om man oversetter til norsk eller engelsk :/